angular/base/package.json

```diff
@@ -13,32 +13,32 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "~8.2.14",
-    "@angular/core": "~8.2.14",
-    "@angular/forms": "~8.2.14",
-    "@angular/platform-browser": "~8.2.14",
-    "@angular/platform-browser-dynamic": "~8.2.14",
-    "@angular/router": "~8.2.14",
+    "@angular/common": "~9.1.6",
+    "@angular/core": "~9.1.6",
+    "@angular/forms": "~9.1.6",
+    "@angular/platform-browser": "~9.1.6",
+    "@angular/platform-browser-dynamic": "~9.1.6",
+    "@angular/router": "~9.1.6",
     "@ionic-native/core": "^5.0.7",
     "@ionic-native/splash-screen": "^5.0.0",
     "@ionic-native/status-bar": "^5.0.0",
     "@ionic/angular": "^5.0.0",
     "core-js": "^2.5.4",
```

Comment:
Is it possible to upgrade to core-js@3 to get rid of the warning?
https://stackoverflow.com/questions/59420896/npm-warn-deprecated-core-js2-6-11-core-js3-is-no-longer-maintained-and-not-r

Reply:
actually able to remove this all together